packages/devtools_app/lib/src/screens/vm_developer/vm_service_private_extensions.dart

```diff
@@ -395,3 +398,84 @@ class ObjectStore {
 
   final Map<String, ObjRef> fields;
 }
+
+/// A `ProfileCode` contains profiling information about a Dart or native
+/// code object.
+///
+/// See [CpuSamples].
+class ProfileCode {
+  ProfileCode({
+    this.kind,
+    this.inclusiveTicks,
+    this.exclusiveTicks,
+    this.code,
+  });
+
+  ProfileCode._fromJson(Map<String, dynamic> json) {
+    kind = json['kind'] ?? '';
+    inclusiveTicks = json['inclusiveTicks'] ?? -1;
+    exclusiveTicks = json['exclusiveTicks'] ?? -1;
```

Comment:
are these values ever going to be used in a summation calculation? if so, having them be -1 may mess up calculations

Reply:
No, these are here simply because `ProfileFunction` in `package:vm_service` has this structure.